maintainers/maintainer-list.nix

```
@@ -7551,6 +7551,12 @@
     githubId = 395821;
     name = "rht";
   };
+  rhoriguchi  = {
```

Please put this into a separate commit with the message ``maintainers: add rhoriguchi``.

```suggestion
  rhoriguchi = {
```